script/bash_images

```diff
@@ -103,11 +103,6 @@ function copy_file_to_server {
     path=${image_server_data[$server,path]}
     scp $image_root/$file $path/$file || die "Failed to transfer $file to $path."
     ;;
-  "s3")
-    cmd=$app_root/script/s3cp.rb
-    wait_for $cmd
-    $cmd $image_root/$file $server $file || die "Failed to transfer $file to $path."
-    ;;
```

Comment:
I see one additional "s3" case in this file for "read_server_directory".

Reply:
Good catch, thanks. I'll take care of it. It wasn't deliberate